src/redteam/index.ts

```diff
@@ -191,18 +268,33 @@ export async function synthesize({
   );
   if (strategies.length > 0) {
     const totalPluginTests = plugins.reduce((sum, p) => sum + (p.numTests || 0), 0);
+
+    console.log(
+      `Strategies: ${strategies.length} totalPluginTests: ${totalPluginTests} ${
+        Object.keys(multilingualStrategy?.config?.languages ?? {}).length ||
+        DEFAULT_LANGUAGES.length
+      } multilingualStrategy: ${multilingualStrategy?.config?.languages}`,
+    );
     logger.info(
       `Using strategies:\n\n${chalk.yellow(
         strategies
-          .map((s) => `${s.id} (${formatTestCount(totalPluginTests)})`)
+          .map((s) => {
+            const testCount =
+              s.id === 'multilingual'
+                ? totalPluginTests *
+                  (Math.max(strategies.length, 1) *
+                    (Object.keys(s.config?.languages ?? {}).length || DEFAULT_LANGUAGES.length))
+                : totalPluginTests;
+            return `${s.id} (${formatTestCount(testCount)})`;
+          })
           .sort()
           .join('\n'),
       )}\n`,
     );
   }
 
   const totalTests =
-    plugins.reduce((sum, p) => sum + (p.numTests || 0), 0) * (strategies.length + 1);
+    plugins.reduce((sum, p) => sum + (p.numTests || 0), 0) * (strategies.length + 1) * numLanguages;
```

Comment:
there is a comment similar to this in the redteam init cli workflow. Would you mind updating it as well? See https://github.com/promptfoo/promptfoo/blob/main/src/redteam/commands/init.ts#L77-L78